config/rbac/common/kustomization.yaml

```diff
@@ -22,6 +22,7 @@ resources:
   # - servingruntime_editor_role.yaml
   # - servingruntime_viewer_role.yaml
   - modelmesh-service-account.yaml
+  - modelmesh-serving-service-account.yaml
```

Comment:
there is now two service accounts, is the older one for a different purpose?

Reply:
Looks upstream also has this service account(modelmesh) so I prefer to create a new service account that this PR contains.